barcodes_gs1_label/models/stock_production_lot.py

```diff
@@ -12,4 +12,9 @@ def print_gs1_barcode_wizard(self):
         action = self.env["ir.actions.act_window"]._for_xml_id(
             "barcodes_gs1_label.barcodes_lot_option_wizard_view_action"
         )
+        label_config = self.env["barcodes.gs1.label.options.config"].search(
+            [("use_default_label", "=", True)]
+        )
+        if label_config:
+            action["context"] = {"default_label_config_id": label_config.id}
```

Comment:
```suggestion
```
If you use a default in the wizard, you don't need this.
Also, very important, the wizard should be opened from the product templates and product variants